### PR TITLE
Speed up doc action

### PR DESCRIPTION
**Pull request by gcattan**
- Just keep subjects 1 and 7
- Reduce the number of runs
- Reduce datasets for plot hs and sar clustering
- Remove SVC and reduce the number of neighbours from 3 to 2 for KNN in the example with classifiers comparison
- Reduce the navigation_depth from 4 to 2 in sphinx conf
- Add update_path=True, where eegbci.load_data is called.

### CODE REVIEW

File: examples/motor-imagery/plot_augmented.py [hunk 1]
```diff
@@ -72,10 +72,14 @@ def transform(self, X):
 tmin, tmax = 1., 2.
 event_id = dict(hands=2, feet=3)
 subject = 7
-runs = [6, 10, 14]  # motor imagery: hands vs feet
+runs = [6, 10]  # motor imagery: hands vs feet
```
Comment (qbarthelemy) [inline examples/motor-imagery/plot_augmented.py:75]: Display is broken: without run 14, curves of results are flat.

I suggest to keep run 14, but to reduce grid search:

```
param_grid = {
    'augmenteddataset__order': [2, 3, 4],
    'augmenteddataset__lag': [5, 7, 9],
}
```

**Review by qbarthelemy — CHANGES_REQUESTED** [review]

**Comment by AmmarMian** [comment]
Sounds good to me to reduce time !

**Review by qbarthelemy — CHANGES_REQUESTED** [review]

**Comment by qbarthelemy** [comment]
Thx a lot @gcattan !

**Comment by gcattan** [comment]
Thx @qbarthelemy !

**Pull request merged**